todo_app/app.py

```diff
@@ -8,4 +9,11 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template('index.html', items = items)
+
+@app.route('/create-todo', methods=['POST'])
+def create_new_todo():
+    new_todo_title = request.form.get('todo-name')
+    add_item(new_todo_title)
+    return index()
```

Comment:
This is a valid approach for making sure we're not duplicating the `index` code, but it also leads to some slightly odd effects (try refreshing the page after you've just created a to-do!); a nicer approach is to pull in the `redirect` function from Flask, and use that like:
```
    return redirect('/')
```